blueocean-dashboard/src/main/js/components/records.jsx

```diff
@@ -23,7 +23,7 @@ export class PipelineRecord extends Record({
     weatherScore: 0,
 }) {
     isFolder() {
-        return this.numberOfFolders !== null;
+        return this._class === 'io.jenkins.blueocean.service.embedded.rest.PipelineFolderImpl';
```

Comment:
@cliffmeyers @michaelneale This is fine as quick solution. However for UI to show any multi-branch, I mean a developer has its own implementation of multi-branch, how does that show up in the UI? For that, UI should really be checking against interface, in this case: `io.jenkins.blueocean.rest.model.BlueMultiBranchPipeline`. That is UI core knows about well known interfaces it supports and on receiving a _class, it should call a common utility class that calls /rest/classes/:classId API to get list of other supported classes/capabilities. Something like: supportsCapability(givenClass, expectedClass), returns boolean. See [Class of resource discovery](https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-rest/README.md#class-of-resource-discovery)

So a TODO should be marked or perhaps a ticket to track this.